public/config.js

```diff
@@ -12,4 +12,6 @@ var jujuDashboardConfig = {
   // only be used for superficial updates like logos. Use feature detection
   // for other environment features.
   isJuju: false,
+  // Show WebCli component if enabled
```

Comment:
Is this accurate? Is it more like force display the WebCLI even if natively unsupported.

Reply:
The WebCLI has a natively unsupported state. I'll update the comment to explicitly state it's for dev purposes.